index.html

```diff
@@ -0,0 +1,311 @@
+              <h2 class="text">
+                Multi-Post <br />
+                Stories Gain+Glory
+              </h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">
+                Multi-Post <br />
+                Stories Gain+Glory
+              </h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product, feature or website. Look through some
+          of my work and<br />
+          experience! If you like what you see and have a project you need
+          coded, <br />don't hestiate to contact me.
+        </p>
+        <button type="button" class="button">Get My Resume</button>
+
+        <div class="aboutGrid">
+          <div class="tools">
+            <ul class="works">
+              <li>
+                <img src="./Images/icon-languages.png" alt="Languages" />
+              </li>
+              <li class="languages">Languages</li>
+              <li>
+                <ul class="language">
+                  <li>JavaScript</li>
+                  <li>Ruby</li>
+                  <li>HTML</li>
+                  <li>CSS</li>
+                </ul>
+              </li>
+            </ul>
+
+            <ul class="works">
+              <li>
+                <img src="./Images/icon-frameworks.png" alt="Frameworks" />
+              </li>
+              <li class="frameworks">Frameworks</li>
+              <li>
+                <ul class="language">
+                  <li>Bootstrap</li>
+                  <li>Ruby on rails</li>
+                  <li>RSpec</li>
+                  <li>Capybara</li>
+                  <li>Selenium</li>
+                </ul>
+              </li>
+            </ul>
+
+            <ul class="works">
+              <li>
+                <img src="./Images/icon-skills.png" alt="Skills" />
+              </li>
+              <li class="skills">Skills</li>
+              <li>
+                <ul class="language">
+                  <li>Codekit</li>
+                  <li>Github</li>
+                  <li>Codepen</li>
+                  <li>Gitlab</li>
+                  <li>Terminal</li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact">
```

Comment:
- [ ] In implementing the Desktop view of a responsive website, in the starting screen with `768px` all elements should display nicely without any overlapping or hidden elements. See the image below is your project at that width, and some elements are hidden on the left side. Your page content should remain centralized across the the width from `768px` to `1080px` so that users with any kind of screen size within that range will still see a nice layout. 


![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/84156714/dd525319-b11d-4343-81c0-e92ad436b1e6)